packages/kbn-generate-console-definitions/src/generate_availability.ts

```diff
@@ -18,11 +46,33 @@ export const generateAvailability = (
     stack: DEFAULT_ENDPOINT_AVAILABILITY,
     serverless: DEFAULT_ENDPOINT_AVAILABILITY,
   };
-  if (endpoint.availability.stack?.visibility) {
-    availability.stack = endpoint.availability.stack?.visibility === 'public';
+  // availability is a required property of the endpoint
+  if (!endpoint.availability) {
+    throw new Error('missing availability for ' + endpoint.name);
```

Comment:
Throwing an error here because the script expects all endpoints to have `availability` according to types. An error here will indicate that something changed in the schema.

Reply:
What do you think about adding a test to expect this scenario to throw? 🤔 I've checked the .test.ts file but couldnt find any for this scenario